elements/osism-ipa/static/opt/osism/templates/osism-netplan.yaml.j2

```diff
@@ -17,3 +18,13 @@ network:
         mtu: 9000
         optional: yes
 {% endfor %}
+
+{% elif variant == 'OOB' %}
+  ethernets:
+    eno1:
```

Comment:
Are you sure that you want to hardcode the interface name here? maybe use `osism_interfaces[0]` instead?